tests/experimental/pydantic/test_fields.py

```diff
@@ -35,7 +34,21 @@
         (pydantic.RedisDsn, str),
     ]
     if IS_PYDANTIC_V1
-    else [],
+    else [
+        (pydantic.PositiveInt, int),
```

Comment:
I think this gets to an equivalent state as far as testing goes with what we had in V1.  I removed all the classes that don't exist anymore in pydanticv2, but kept all the ones that do still exist.